_layouts/post.html

```diff
@@ -12,7 +12,7 @@
 </script>
 
 <article class="ph3 pt0 pb4 article">
-  <div class="tc mb4">
+  <div class="tc mb4 db" id="author-blurb">
```

Comment:
Usually, I check tachyons classes from here: https://roperzh.github.io/tachyons-cheatsheet/

If you wish to use max-width, I believe you can use `mw-X`.